packages/bruno-app/src/providers/ReduxStore/slices/collections/actions.js

```diff
@@ -939,7 +939,16 @@ export const createCollection = (collectionName, collectionFolderName, collectio
       .catch(reject);
   });
 };
+export const cloneCollection = (collectionName, collectionFolderName, collectionLocation, perviousPath) => () => {
+  const { ipcRenderer } = window;
 
+  return new Promise((resolve, reject) => {
+    ipcRenderer
+      .invoke('renderer:clone-collection', collectionName, collectionFolderName, collectionLocation, perviousPath)
+      .then(resolve)
+      .catch(reject);
+  });
```

Comment:
I have corrected this

Reply:
Hey, i you need to look at this again. Now the Modal always stays open, because the promise is never resolved.

I think this would make more sense:
```
export const cloneCollection = (collectionName, collectionFolderName, collectionLocation, perviousPath) => () => {
  const { ipcRenderer } = window;

  return ipcRenderer.invoke(
    'renderer:clone-collection',
    collectionName,
    collectionFolderName,
    collectionLocation,
    perviousPath
  );
};
```

Reply:
Sure, now it is working fine